src/App.css

```diff
@@ -116,13 +116,36 @@ button:hover {
   justify-content: space-between;
   padding-left: 1rem;
   padding-top: 2rem;
+  border-bottom: .1rem solid rgb(237, 225, 207);
 }
 
 .cart-x-button {
   margin-right: 1rem;
-  width: 30px;
+  width: 2rem;
+  height: 2rem;
+
+}
+
+.product-info {
+  float: right;
```

Comment:
`float` is good for text mainly. it's not good to be used for multiple components inside of it. It's also unpredictable. 

It's better to use `flex` to have items align properly.